files/bootstrap.sh

```diff
@@ -75,6 +76,11 @@ while [[ $# -gt 0 ]]; do
             shift
             shift
             ;;
+        --containerd-extra-config-toml)
```

Comment:
I would prefer it as you described it, but decided to stick with the way --docker-config-json is implemented to avoid unexpected surprises. If enough EKS folks tell me that the file-pointer way is the preferred one, I'll go with it.